openstudiocore/src/utilities/data/TimeSeries.hpp

```diff
@@ -50,7 +50,7 @@ namespace openstudio{
 
         /// constructor from start date and time, interval length, values, and units
         /// throws if values is empty
-        TimeSeries_Impl(const DateTime& startDateTime, const Time& intervalLength, const Vector& values, const std::string& units);

```

Comment:
@jasondegraw did you ever make that image that showed how to interpret time for all the different constructors?